src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -1058,4 +1058,14 @@ public void sendKeysHumanSpeed(WebElement input, String keys) {
       }
   }
   
+  public void acceptPopupWindow() {
+      Alert alert = driver.switchTo().alert();
+      alert.accept();
```

Comment:
i prefer readibility, so that code reflects user actions:
1. there is an alert popup
2. then user can accept the popup